lang/en.json

```
@@ -559,16 +570,22 @@
         "MatomoConfigurationMatomoDisableCookiesDescription": "Disables all first party cookies.",
         "MatomoConfigurationMatomoRequireConsentTitle": "Require tracking consent",
         "MatomoConfigurationMatomoRequireConsentDescription": "Track only when user gave tracking consent. In a consent screen you need to call \"window._paq=window._paq||[];window._paq.push(['rememberConsentGiven']);\" when the user gives consent.",
-        "MatomoConfigurationMatomoCustomCookieTimeOutEnableTitle": "Enable custom visitor Cookie timeout",
-        "MatomoConfigurationMatomoCustomCookieTimeOutEnableDescription": "Allows you to set a custom time in full days for the _pk_id cookie.",
-        "MatomoConfigurationMatomoCustomCookieTimeOutTitle": "The pk_id expire time in full days",
-        "MatomoConfigurationMatomoCustomCookieTimeOutDescription": "Timeout in full days. By default, the Matomo tracking cookie expires in 13 months (365 + 28 days) = 393 days.",
+        "MatomoConfigurationMatomoCustomCookieTimeOutEnableTitle": "Enable Custom Cookie Timeouts",
```

@EPNW-Eric Since this keys are already translated, you should create a new one rather than updating the current one, something like below or any name you think is correct
```suggestion
        "MatomoConfigurationMatomoCustomVisitorCookieTimeOutEnableTitle": "Enable Custom Cookie Timeouts",
```

---

The key is just used once, so changing the content would not affect any other places in the UI. This also means that if we add a new key, this old key is never used anywhere again and just obsolete. What if we keep the update-approach but I delete the key from all other language files?

If a new-key-approach is preferable to this, I think `MatomoConfigurationMatomoCustomCookieTimeOutsEnableTitle` would fit, even if it just differs from the old key by the plurals *s* in *TimeOuts*. Would that be fine or is it too confusing?

---

if you search for `MatomoConfigurationMatomoCustomCookieTimeOutEnableTitle` you would notice that its already translated in other languages,  and it won't get translated so that's why we are asking you to use new keys and yes you can delete the old text from `en.josn` if it is not being used anywhere else